.gitignore

```diff
@@ -6,3 +6,5 @@ logs/
 *.beam
 variables-ct*
 *.iml
+rebar.lock
```

Comment:
I think the `rebar.lock` should be checked into the version control repo.

Reply:
I don't think rebar.lock should be checked into the version control repo.